lnbits/wallets/base.py

```diff
@@ -60,7 +60,7 @@ def create_invoice(
         pass
 
     @abstractmethod
-    def pay_invoice(self, bolt11: str) -> Coroutine[None, None, PaymentResponse]:
+    def pay_invoice(self, bolt11: str, fee_limit_msat: int) -> Coroutine[None, None, PaymentResponse]:
```

Comment:
I suggest adding this comment above `def pay_invoice`:

```python
# WARNING: correct handling of fee_limit_msat is required to avoid security vulnerabilities!
# The backend MUST NOT spend satoshis above invoice amount + fee_limit_msat.
```